src/character.py

```diff
@@ -45,10 +55,26 @@ def main():
     clock = pygame.time.Clock()
 
     running = True
+    jumping = 0
+    jumping_counter = 0
     while running:
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 running = False
+        keys = pygame.key.get_pressed()
+        if keys[pygame.K_UP]:
+            jumping = True
+        if jumping:
+            if jumping_counter >= 3:
```

Comment:
Are there built-in functions for this in pygame? Not sure if there are, but might be something to look into.